StandaloneMmPkg/Include/Protocol/PiMmCpuDriverEp.h

```diff
@@ -0,0 +1,54 @@
+/** @file
+  This file describes the PiMm Cpu Driver Entry Point Protocol.
+  the protocol is for defining handler for mm communication request event
+  according to cpu driver.
+
+  Copyright (c) 2024, Arm Limited. All rights reserved.<BR>
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+
+**/
+
+#ifndef PI_MM_CPU_DRIVER_EP_H_
+#define PI_MM_CPU_DRIVER_EP_H_
+
+#include <PiMm.h>
+
+#define PI_MM_CPU_DRIVER_EP_GUID  { \
+  0x6ecbd5a1, 0xc0f8, 0x4702, { 0x83, 0x01, 0x4f, 0xc2, 0xc5, 0x47, 0x0a, 0x51 } \
+  }
+
+/**
+  The PI Standalone MM entry point for handling mm communication request
+  according to cpu driver.
+  At first, mm communication request is delivered via smc to DelegatedEventLoop.
+  In DelegatedEventLoop, after SMC call sanity checking, it calls this function
+  to further handling via MmiHandler based on mm communication protocol which
+  saved in CommBufferAddr.

```

Comment:
Can you add more comments to explain how this protocol is used in the ARM MMI flow? e.g.:
1. StandaloneMmCore dispatches all MM drivers including ARM's StandaloneMmCpu driver
2. StandaloneMmCpu driver declares its MMI CPU Entry Point through the new protocol
3. StandaloneMmCore gets the MMI CPU entry point by locating the protocol after all drivers are dispatched.
4. DelegatedEventLoop calls the MM CPU entry point.

Reply:
Thanks. I'll describe more detail about it.